packages/happo-target-firefox/src/__tests__/checkBrowserVersion-test.js

```diff
@@ -2,7 +2,7 @@ jest.mock('child_process');
 const childProcess = require('child_process');
 
 jest.mock('../config');
-const config = require('../config');
+const { config } = require('happo-core');
```

Comment:
It looks like this fixes a bug as well. I did the same fix in f88d3bb9cb8ef, and unless it's a total pain, maybe we should rebase this PR/branch on top of that as well. 

EDIT - I think rebasing might end up being more of a hassle than it's worth. You should update the `jest.mock` so that the right module is mocked though.